### PR TITLE
Add Code of Conduct

### PR DESCRIPTION
**Pull request by haya14busa**

### CODE REVIEW

**Comment by haya14busa** [comment]
I created a google mailing list (https://groups.google.com/forum/#!forum/reviewdog) as a contact in the code of contact.

I don't plan to use the mailing list for other purposes for now, but please feel free to join.

**Comment by LaurenceGA** [comment]
> I created a google mailing list (https://groups.google.com/forum/#!forum/reviewdog) as a contact in the code of contact.
> 
> I don't plan to use the mailing list for other purposes for now, but please feel free to join.

Don't have permission to access group when opening link. Might be misconfigured :thinking:

**Comment by haya14busa** [comment]
Thanks! @LaurenceGA 

I updated the configuration. I guess anyone can request to join the group now. https://groups.google.com/forum/#!forum/reviewdog

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```
Comment (LaurenceGA) [inline CODE_OF_CONDUCT.md:1]:
```suggestion
# Reviewdog Code of Conduct
```
or maybe
```suggestion
# Reviewdog community Code of Conduct
```
I looked at [their examples](https://www.contributor-covenant.org/adopters/) and it seems like a lot of people leave `Contributor Covenant` in, but I don't think their meant to...? Seems weird... Quite a few people replace it too.
There's already an `Attribution` section at the bottom, so I think it's obvious where it comes from.

**Comment by haya14busa** [comment]
Thank you for your review, @LaurenceGA!

> What about adding a badge to the readme?

Added. I also update the Contributor Covenant to version 2.0.
GitHub suggested to use 1.4 by default, but I think we're ok to use version 2.0.

**Pull request merged**